detox/src/ios/expectTwo.js

```diff
@@ -154,6 +154,11 @@ class Element {
     return this.withAction('clearText');
   }
 
+  accessibilityAction(actionName) {
```

Comment:
The API should be named `performAccessibilityAction()`

Reply:
@LeoNatan Just in this place or everywhere I've created a method named `accessibilityAction`?

I had considered  `performAccessibilityAction` originally and passed on it 🤦

Reply:
I only care about the public API, accessible to Detox users (and docs). The internal stuff doesn't matter, it's descriptive enough.

Reply:
Thanks